doc-surrealdb_versioned_docs/version-1.x/surrealql/parameters.mdx

```diff
@@ -7,19 +7,51 @@ description: Parameters can be used like variables to store a value which can th
 
 # Parameters
 
-Parameters can be used like variables to store a value which can then be used in a subsequent query. A parameter can store any value, including the result of a query. Parameters can be defined within the SQL, or can be passed in using the client libraries as request variables.
+Parameters can be used like variables to store a value which can then be used in subsequent queries. To define a parameter in SurrealQL, use the [`LET`](../surrealql/statements/let) statement. The name of the parameter should begin with a `$` character.
 
 ## Defining parameters within SurrealQL
 
-To define a parameter in SurrealQL, use the [`LET`](../surrealql/statements/let) statement. The name of the parameter should begin with a `$` character.
-
 ```surql
 -- Define the parameter
-LET $name = "tobie";
+LET $surname = "Morgan Hitchcock";
```

Comment:
thought: seems a little confusing, why not splitting with `$middle_name` and `$last_name`?

Reply:
Hmm, how about $suffix or something else?